tools/ConfigureGeneratedApplications/Model/Replacement.cs

```diff
@@ -5,6 +5,11 @@ namespace ConfigureGeneratedApplications.Model
 {
     public class Replacement
     {
+        public Replacement()
```

Comment:
In .NET core 3.1, System.Text.Json was less clever, and needed a public parameterless constructor